library/std/src/sys/fs/uefi.rs

```diff
@@ -92,8 +97,15 @@ impl FilePermissions {
 }
 
 impl FileTimes {
-    pub fn set_accessed(&mut self, _t: SystemTime) {}
-    pub fn set_modified(&mut self, _t: SystemTime) {}
+    pub fn set_accessed(&mut self, t: SystemTime) {
+        self.accessed =
+            t.to_uefi(self.accessed.timezone, self.accessed.daylight).expect("Invalid Time");
```

Comment:
Can we avoid this panic? As the `SystemTime` was created successfully (i.e. `checked_add` did not return an error), this panic will come as a surprise to users.

A better way perhaps would be to try the initial timezone first, and if that fails, return the closest timezone (minute offset) that still allows the time to be represented. If filesystems ignore the timezone field, this would lead to the first and last 1440 minutes being collapsed into one minute, which doesn't seem too bad.